.repo-metadata.json

```diff
@@ -6,6 +6,7 @@
   "issue_tracker": "https://issuetracker.google.com/savedsearches/559753",
   "release_level": "ga",
   "language": "python",
+  "library_type": "GAPIC_COMBO",
```

Comment:
Are there handwritten helpers in this library?

Reply:
Good catch! Fixed.